### DISCUSSION TITLE
加群事件

### DISCUSSION THREAD

**Discussion by Sjxixi-y** [simple-robot/Discussions#166] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

如何获取详细的加群者信息？我想获取等级来进行一个简单的判断。

**Comment by ForteScarlet** [comment, marked as answer]
参考 [mirai#1483](https://github.com/mamoe/mirai/issues/1483) ，可以使用 `MiraiBot.originalBot` 和申请者的ID来查询申请者的用户信息。

**Reply by Sjxixi-y** [reply]
已解决
